lib/gym/runner.rb

```diff
@@ -9,7 +9,9 @@ class Runner
     def run
       clear_old_files
```

Comment:
in the case of use of `:use_legacy_build_api` option, the `clear_old_files` will delete the previously built archive. So the export only will then fail. I suspect the `clear_old_files` should move into the `unless` block as well